VocaDbWeb/Scripts/Repositories/SongListRepository.ts

```diff
@@ -72,6 +78,10 @@ module vdb.repositories {
 
 		};
 
+		public getTagSuggestions = (songId: number, callback: (contract: dc.tags.TagUsageForApiContract[]) => void) => {
+			$.getJSON(this.urlMapper.mapRelative("/api/songLists/" + songId + "/tagSuggestions"), callback);

```

Comment:
I didn't see this method in SongListApiController. Are you intending to add tag suggestions based on songs in the list?